### ISSUE TITLE
Unable to reproduce the results of ET on QM9

### ISSUE THREAD

**Issue by 20171130** [torchmd/torchmd-net#54]
I am trying to reproduce the results in the paper using equivariant transformers on QM9 to predict the energy. Here is my config, the hyperparameters are the same as those mentioned in appendix A in "EQUIVARIANT TRANSFORMERS FOR NEURAL NETWORK-BASED MOLECULAR POTENTIALS".
```
activation: silu
aggr: add
atom_filter: -1
attn_activation: silu
batch_size: 128
coord_files: null
cutoff_lower: 0.0
cutoff_upper: 5.0
dataset: QM9
dataset_arg: energy_U0
dataset_root: ~/data
derivative: false
distance_influence: both
distributed_backend: ddp_cpu
early_stopping_patience: 300
ema_alpha_dy: 1.0
ema_alpha_y: 1.0
embed_files: null
embedding_dimension: 256
energy_files: null
energy_weight: 1.0
force_files: null
force_weight: 1.0
inference_batch_size: 128
load_model: null
log_dir: /tmp/logs
lr: 0.0004
lr_factor: 0.8
lr_min: 1.0e-06
lr_patience: 15
lr_warmup_steps: 10000
max_num_neighbors: 32
max_z: 100
neighbor_embedding: true
ngpus: 1
num_epochs: 300
num_heads: 8
num_layers: 8
num_nodes: 1
num_rbf: 64
num_workers: 4
output_model: Scalar
precision: 32
prior_model: Atomref
rbf_type: expnorm
redirect: false
reduce_op: add
save_interval: 3
seed: 3
splits: null
standardize: false
test_interval: 3
test_size: null
train_size: 110000
trainable_rbf: false
val_size: 10000
weight_decay: 0.0
```

However, I was unable to reproduce the results in the paper. 
1. The model I constructed only has 2.5 M parameters, while according to appendix A, the model in the paper with the same hyperparameters has 6.87M parameters.
2. The best test error I got was 12meV, which is significantly higher than 6.24meV reported in the paper.

**Comment by PhilippThoelke** [comment]
I just added a set of config files and pretrained models to the repo, which contain the hyperparameter sets you can use to train the model on QM9, MD17 and ANI1 as reported in the paper. I also added a readme file to the examples directory with some instructions on training and loading models.

The hyperparameters you used mostly look good, however, you are missing a key argument. By default the training script will select a SchNet-like graph network, which explains the difference in accuracy and parameter count. You should get correct results if you add `model: equivariant-transformer` to you config file or simply use [this](https://github.com/torchmd/torchmd-net/blob/main/examples/ET-QM9.yaml).
